### ISSUE TITLE
The Open menu is greyed out and unselectable

### ISSUE THREAD

**Issue by owldown** [DavidPhillipOster/ThumbHost3mf#12]
Just installed and gave permission to the Quicklook extension. When I try to open a folder with the app, the Open menu item is greyed out.

**Comment by DavidPhillipOster** [comment]
Thanks - I was dropping .3mf and .gcode files on the app icon in the dock, or dragging them to the window. You've found a real bug. It should just be in the app, though, and not in how the Finder interacts with the plugin.

**Comment by owldown** [comment]
I was trying to follow the directions:

> Use ThumbHost3MF's File > Open menu item and point at a directory containing .gcode or .3mf files: that will kick the Finder into noticing the embedded thumbnail presenter.

Dragging files and folders does work for me, so this is great! Now I just need to find a way to *generate* thumbnails for the tons of files I have that don't have embedded thumbnails.

**Comment by DavidPhillipOster** [comment]
Once the Finder (Actually the Quicklook system) picks up the plugin, the Finder's icon view should get the correct icons for files that have thumbnails without any additional action on your part.

I used a Finder search to see my .3mf files, so it was easy to find the ones that had no thumbnails.

For .3mf files that don't have thumbnails, at least in PrusaSlicer, I just opened an old .3mf, verified in the Printer Settings panel that opening the file had not overridden the default options for writing thumbnails, and **Save As** back over the older original. After the first few, it was just open the file, and save as immediately.
